nixos/tests/all-tests.nix

```diff
@@ -361,6 +361,7 @@ in {
   man = handleTest ./man.nix {};
   mariadb-galera = handleTest ./mysql/mariadb-galera.nix {};
   mastodon = handleTestOn ["x86_64-linux" "i686-linux" "aarch64-linux"] ./web-apps/mastodon.nix {};
+  mastodon-ext = handleTestOn ["x86_64-linux" "i686-linux" "aarch64-linux"] ./web-apps/mastodon-ext.nix {};
```

Comment:
It would be nicer to put both mastodon tests in a directory and then use `discoverTests`

Reply:
How to combine `handleTestOn` + `discoverTests` ?

Reply:
362|   mariadb-galera = handleTest ./mysql/mariadb-galera.nix {};
          363|   mastodon = discoverTests (import ./web-apps/mastodon {});
             |                             ^
          364|   matomo = handleTest ./matomo.nix {};

Reply:
@turion can we keep current variant? For example, all nginx tests are not combined into one test either.

Reply:
Fixed.